packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -57,14 +58,16 @@ class BouncingScrollSimulation extends Simulation {
         _springTime = _frictionSimulation.timeAtX(trailingExtent);
         _springSimulation = _overscrollSimulation(
           trailingExtent,
-          math.min(_frictionSimulation.dx(_springTime), maxSpringTransferVelocity),
+          math.min(
+              _frictionSimulation.dx(_springTime), maxSpringTransferVelocity),
```

Comment:
fyi we don't use `dartfmt` in this repo (because it does things like this) -- see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo